### ISSUE TITLE
TwitterReverseAuth brings a dependency on AFNetworking 1.x

### ISSUE THREAD

**Issue by orta** [artsy/Artsy-Authentication#12]
```
   Elasticity   master ✘ ✹  bundle exec pod install 

Analyzing dependencies
Pre-downloading: `Artsy+Authentication` from `https://github.com/artsy/Artsy_Authentication.git`
[!] Unable to satisfy the following requirements:

- `AFNetworking` required by `Podfile`
- `AFNetworking (= 2.4.1)` required by `Podfile.lock`
- `AFNetworking (~> 1.3.4)` required by `TwitterReverseAuth (1.0)`
```

**Comment by ashfurrow** [comment]
[This](https://github.com/AFNetworking/AFOAuth1Client/pull/115) could obviate this problem.

**Comment by orta** [comment]
Eigen is now using that fork in my AFNetworking 2 fork

**Comment by orta** [comment]
I've made a no-twitter branch, https://github.com/artsy/Artsy_Authentication/tree/no_twitter to obviate this problem for me ATM.

**Comment by orta** [comment]
Might be an option: https://github.com/webparadox/LetMeAuth-iOS

**Comment by orta** [comment]
:tada: closed by #14
